src/main/java/cern/molr/sample/RunnableMole.java

```diff
@@ -27,7 +26,7 @@
  * @author nachivpn
  * @see Mole
  */
-public class RunnableMole implements Mole<Null,Null> {
+public class RunnableMole implements Mole<Void,Void> {
```

Comment:
Yes, you're right - but the current implementation of `mole.run(
)` doesn't use any arguments for constructing the Runnable instance or elsewhere in it's body